assets/magick-js.php

```diff
@@ -6,5 +6,7 @@
   header('Expires: ' . gmdate('D, d M Y H:i:s',time()+604800) . ' GMT');
   header("Content-type: text/javascript; charset=utf-8");
   readfile('bootstrap.bundle.min.js');
+  echo(PHP_EOL);
+  readfile('cli-translate.min.js');

```

Comment:
Here is where I add the cli-translate.min.js
I had to add the PHP_EOL because the bootstrap.bundle.min.js ends with a comment line and not a newline.

Reply:
I think I would rather have the `sourceMappingURL` removed from those bootstrap files.

Reply:
Ah, I see now. 

//# sourceMappingURL=bootstrap.bundle.min.js.map